x-pack/plugins/security_solution/public/common/components/endpoint/page_view.tsx

```diff
@@ -24,7 +24,7 @@ import { EuiTabProps } from '@elastic/eui/src/components/tabs/tab';
 
 const StyledEuiPage = styled(EuiPage)`
   &.endpoint--isListView {
-    padding: 0;
+    padding: 0 70px 0 24px;
```

Comment:
updated for 2 reasons, to be more uniform with the rest of the security app and to allow functional tests to execute properly as the overlap with the timeline was blocking click events